docs/config.toml

```diff
@@ -2,74 +2,563 @@ baseurl = "https://docs.helm.sh/"
 languageCode = "en-us"
 title = "Helm Docs"
 theme = "helmdocs"
+
+contentDir = "source/docs"
 publishdir = "app"
 
 [params]
     title = "Helm Docs"
     author = "Ronan Flynn-Curran"
     description = "Documentation for Helm - The Kubernetes Package Manager."
 
-canonifyURLs = "false"
-relativeURLs = "true"
+canonifyURLs = "true"
+relativeURLs = "false"
 pluralizeListTitles = "false"
 plainIdAnchors = "true"
+defaultContentLanguage = "en"
+
 
 [[menu.main]]
+  name = "Home"
+  url = "https://docs.helm.sh"
```

Comment:
perhaps related to comment above -- are we sure we want to hardcode the url here? (as opposed to it being relative/based on environment where deployed)